generators/app/templates/javascript/gulpfile.js

```diff
@@ -6,7 +6,7 @@ const nodemon = require('gulp-nodemon');
 gulp.task('watch', () =>
   nodemon({
     script: 'server.js',
-    watch: ['config/*', 'services/*', 'skill/*', 'server.js'],
+    watch: ['src/*', 'server.js'],
```

Comment:
yes, check how we're using nodemon in other skills, no need for gulpfile, unless we also want to include the S3 syncing functionality we have in most skills